components/org.wso2.carbon.identity.api.server.organization.management/org.wso2.carbon.identity.api.server.organization.management.v1/src/gen/java/org/wso2/carbon/identity/api/server/organization/management/v1/model/BasicOrganizationResponse.java

```diff
@@ -77,6 +81,31 @@ public BasicOrganizationResponse id(String id) {
         this.id = id;
         return this;
     }
+
+    public BasicOrganizationResponse attributes(List<Attribute> attributes) {
```

Comment:
Are these manually changed?
The files under /gen folder should not be updated manually.
Those are generated from the added plugin in pom based on the yaml.
Can't find the yaml changes in the PR

Reply:
Apologies for the error. The /gen folder was manually updated. I've corrected this by updating the YAML file.